qiskit/transpiler/passes/optimization/commutative_optimization.py

```diff
@@ -0,0 +1,70 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2025.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Commutative Optimization transpiler pass."""
+
+
+from qiskit.transpiler.basepasses import TransformationPass
+from qiskit.circuit.commutation_library import SessionCommutationChecker as scc
+
+from qiskit._accelerate import commutative_optimization
+
+from qiskit.transpiler.passes.utils.control_flow import trivial_recurse
+
+
+class CommutativeOptimization(TransformationPass):
+    """
+    Cancel/merge gates exploiting commutativity relations.
+
+    The pass will:
+
+    * Cancel pairs of inverse gates.
+    * Cancels pairs of gates that are inverse up to a global phase (adjusting
+      the global phase accordingly).
+    * Combines different types of RZ-rotations into an RZ-gate.
+    * Combines different types of RX-rotations into an RX-gate.
+
+    This pass generalizes both :class:`.CommutativeCancellation` and
+    :class:`.CommutativeInverseCancellation` transpiler passes.
+    """
+
+    def __init__(self, approximation_degree: float = 1.0, matrix_max_num_qubits: int = 0):
+        """
+        Args:
+            approximation_degree: Used in the tolerance computations.
```

Comment:
Could you clarify that this refers to the threshold used in the average gate fidelity, which is used to check if gates cancel?